docs/integration.md

```diff
@@ -0,0 +1,435 @@
+        <version>x.x.x</version>
+        <scope>runtime</scope>
+    </dependency>
+    <dependency> <!-- (3)! -->
+        <groupId>com.my.company</groupId>
+        <artifactId>chutney-extensions</artifactId>
+        <version>x.x.x</version>
+        <scope>runtime</scope>
+    </dependency>
+    ```
+
+    1. Required if using SQL actions in scenarios which execute requests against an Oracle database.
+    2. Required if using JMS actions in scenarios which read JMS messages from a weblogic server.
+    3. Some external artifact with Chutney actions or functions extensions to be used in scenarios.
+
+
+# Configuration
+
+Beside java dependencies, we have to package configuration files in the project to be able to start properly the application. Configuration set some [Spring Boot](https://docs.spring.io/spring-boot/docs/2.7.x/reference/html/application-properties.html#appendix.application-properties) and [Chutney](#chutney-specifics) properties as well as logging setup.
+
+Below YAML examples should then be viewed as included in the default Spring Boot configuration file **application.yml**.
+
+!!! warning "Handling secrets"
+    We do not discuss here how one could handle secrets present in configuration files because this could vary extensively on the deployment process applied.
+    
+    For example, if [Ansible](https://docs.ansible.com/ansible/latest/index.html) is used, one could package a subset of configuration files to let them be filtered at deployment time and be included in the runtime classpath of the application. 
+
+## Database
+
+Chutney relies internally on a relational database schema embedded and abstracted by [Liquidbase](https://www.liquibase.org/).
+
+Associated changelog could be found [here](https://github.com/chutney-testing/chutney/blob/master/server/src/main/resources/changelog/db.changelog-master.xml).
+
+!!! note
+    **Chutney has been tested with H2 and PostgreSQL databases.**
+
+The configuration of the associated datasource is done by setting **spring.datasource** properties:
+=== "H2 (memory)"
+    ``` yaml
+    spring:
+        datasource:
+            url: jdbc:h2:mem:dbName
+    ```
+    !!! note
+        The module [local-dev](https://github.com/chutney-testing/chutney/tree/master/packaging/local-dev) is an example of an H2 embedded configuration with filesystem persistence.
+
+=== "PostgreSQL (SSL two way)"
+    ``` yaml
+    spring:
+        datasource:
+            url: jdbc:postgresql://host:port/dbName?ssl=true&sslmode=verify-ca&sslfactory=org.postgresql.ssl.DefaultJavaSSLFactory&currentSchema=mySchema
+            username: user
+    ```
+
+## Authentication & Authorization
+
+!!! important
+    The module [local-dev](https://github.com/chutney-testing/chutney/tree/master/packaging/local-dev) presents an example including :
+
+    * Use of the **mem-auth** profile.
+    * An example of a custom LDAP authentication (the LDAP server is an embedded one).
```

Comment:
Maven module [local-dev](https://github.com/chutney-testing/chutney/tree/master/packaging/local-dev) shows :

    * How to use in memory authentication and roles, see the **mem-auth** profile.
    * How to use a custom LDAP authentication (for example purpose, it uses an embedded LDAP server).